ffs/integrationtest/integration_test.go

```diff
@@ -1468,6 +1472,22 @@ func requireCidConfig(t *testing.T, fapi *api.API, c cid.Cid, config *ffs.CidCon
 	require.Equal(t, *config, currentConfig)
 }
 
+func requireStorageDealRecord(t *testing.T, fapi *api.API, _ cid.Cid) {
+	time.Sleep(time.Second)
+	recs, err := fapi.ListStorageDealRecords(deals.WithIncludeFinal(true))
+	require.Nil(t, err)
+	require.Len(t, recs, 1)
+	// ToDo: this should be PayloadCid once we can get it
+	// require.Equal(t, c, recs[0].DealInfo.PieceCID)
```

Comment:
Need to sort out the data cid thing. For now, just check that a record is returned.

Reply:
Can't this be enabled checking RootCid?

Reply:
`StorageDealInfo` has no `RootCid`.

Reply:
Only `PieceCID` and `PproposalCid`

Reply:
Sorry, I meant that `Store` could use the `dataCid` value in the `recordDeal` to keep track of that.
Is that what you're planning?

Reply:
Yea, something like that should work. Actually, I'll just try to get that in now.

Reply:
Ok done. I'm storing `RootCid` on `StorageDealRecord` so that information is available when someone queries for storage deal records. Other apis that give deal status like `Watch` still just use `StoreDealInfo` which is derived directly from a Lotus `DealInfo` and doesn't not include root cid. That would be a good improvement in Lotus to include that information.